hooks/playbooks/ipa-post-deploy.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Run federation setup on openstack post reproducer deploy
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: true
```

Comment:
almost sure, that gather_facts might not be required

Reply:
looks like https://gitlab.cee.redhat.com/ci-framework/ci-framework-testproject/-/merge_requests/1274 is failing, but with an unrelated error